scalameta/semantic/shared/src/main/scala/scala/meta/internal/semantic/vfs/Paths.scala

```diff
@@ -1,28 +1,26 @@
 package scala.meta.internal.semantic.vfs
 
 import scala.meta.internal.io.PathIO.fileSeparator
+import scala.meta.io.AbsolutePath
 import scala.meta.io.RelativePath
 import org.scalameta.invariants.require
 
 object Paths {
-  private val semanticDbPrefix = "META-INF" + fileSeparator + "semanticdb" + fileSeparator
+  private val semanticDbPrefix: RelativePath = RelativePath("META-INF").resolve("semanticdb")
   private val semanticDbSuffix = ".semanticdb"
   private val scalaPrefix = ""
   private val scalaSuffix = ".scala"
 
   def isSemanticdb(path: RelativePath): Boolean = {
-    path.toString.startsWith(semanticDbPrefix) &&
-    path.toString.endsWith(semanticDbSuffix)
+    path.toNIO.startsWith(semanticDbPrefix.toNIO) &&
+    path.path.getFileName.toString.endsWith(semanticDbSuffix)
```

Comment:
Also, I'm not sure whether `path.path.getFileName.toString` constitutes an improvement with respect to `path.toString`. Can you comment on this?